tests/test_banneduser_experiment_controller.py

```diff
@@ -264,16 +266,43 @@ def test_get_account_age(self, seconds_ago, want, experiment_controller):
         ],
     )
     def test_get_condition(self, seconds_ago, want, experiment_controller):
-        # NOTE: Condition is currently the same value as `_get_account_age`.`
+        # NOTE: Condition is currently the same value as `_get_account_age_bucket`.
+        now = datetime.datetime.utcnow().timestamp()
+        condition = experiment_controller._get_condition(now, now - seconds_ago)
+        assert condition == want
+
+    @pytest.mark.parametrize(
+        "seconds_ago,want",
+        [
+            (1, True),
+            (864001, False),
+        ],
+    )
+    def test_is_too_new(self, seconds_ago, want, experiment_controller):
+        # NOTE: Condition is currently the same value as `_get_account_age_bucket`.
         now = datetime.datetime.utcnow().timestamp()
-        condition = experiment_controller._get_condition(now - seconds_ago)
+        condition = experiment_controller._is_too_new(now, now - seconds_ago)
         assert condition == want
 
     def test_assign_randomized_conditions(self, modaction_data, experiment_controller):
+        assert len(experiment_controller._previously_enrolled_user_ids()) == 0
+
+        now = datetime.datetime.utcnow().timestamp()
         user_modactions = experiment_controller._find_eligible_newcomers(modaction_data)
-        experiment_controller._assign_randomized_conditions(user_modactions)
+        experiment_controller._assign_randomized_conditions(now, user_modactions)
+
         assert len(experiment_controller._previously_enrolled_user_ids()) > 1
 
+    def test_assign_randomized_conditions_exclusion(
```

Comment:
Perhaps we could rename this to `test_assign_randomized_conditions__new_accounts_exclusion` or something, to clarify that this isn't actually testing 'exclusion' regarding randomized conditions?